content/md/products-super-power/enrich-products-variants.md

```diff
@@ -53,6 +53,12 @@ If you are looking for clothing with the color "red" and the size "S", the red S
 
 ![Search results with color red and size S](../img/Products_ProductModel_Search3.png)
 
+:::info
+_Since 2.3_ if you search by categories, the grid will automatically display the product models or the products classified according to the category selected in the grid.
+If a product model is not classified in the selected category but its children products are, the products will be displayed in the grid.
+So all the filters, attributes and categories selected in the grid, will be taken into account to display the level you would like to enrich.
```

Comment:
Maybe we could add that, if the selected filters wcould display a product model and its variant, the product grid will always display the variant product.

Reply:
No, really it displays the upper level so the product model, it's my previous example on the red color, the sub product model is red, its variant products are red so we display the sub product model red, but I add some details.